usaspending_api/etl/tests/integration/test_load_to_from_delta.py

```diff
@@ -153,9 +158,16 @@ def load_delta_table_from_postgres(
     if alt_name:
         cmd_args += [f"--alt-name={alt_name}"]
 
+    create_cmd_args = copy.deepcopy(cmd_args)
+    load_cmd_args = copy.deepcopy(cmd_args)
+
+    if incremental:
```

Comment:
Is `incremental` a Boolean? If not I think this if statement could be made more clear.

Reply:
Yes, this is a boolean